### PR TITLE
Blog: Looking Back at the LFX Mentorship Program Spring '21

### PR DESCRIPTION
**Pull request by Jibbow**
Signed-off-by: Andreas Zimmerer <andreas.zimmerer@tum.de>

### CODE REVIEW

**Comment by milasuperstar** [comment]
@WT-Liu PTAL 👀

File: content/blog/lfx-2021-copr-v2.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+While this was certainly a plus when I applied to TiKV, it is by no means required! The LFX Mentorship Program is a perfect start into the world of open-source software and the community around it :)
+
+Oftentimes I hear from other students that it's hard to get started with open-source development; sometimes people don't feel that they have the required skills or knowledge.
+You might not be familiar with all technologies being used in a project or you don't feel you are proficient enough with the main programming language of the project.
+But fear not: don't block yourself by finding unnecessary excuses, just start!
+Contributing to open-source projects is the *perfect* opportunity to learn new technologies and get better at programming because you'll receive valuable feedback that you will not find easily elsewhere.
+Start small by reporting bugs or other issues. If you are up for a challenge, ask the maintainers if you could fix a certain issue with their help.
```
[the remark this answers is not in the document]
Comment (WT-Liu) [reply content/blog/lfx-2021-copr-v2.md:24]: For TiKV project, a newbie could ask reviewers, committers, sig leaders or maintainers for help. Each project has their own governance so I think it is safe to keep it "project maintainers" here. As for TiKV mentorship programs, mentors are maintaining and tracking everything so I suggest changing "maintainers" into "mentors".

**Review by WT-Liu — APPROVED** [review]
Some minor suggestions. The rest looks perfect to me! Thanks for this beautiful blog post!

**Comment by Jibbow** [comment]
@WT-Liu thanks for the review! :) 
PTAL

**Pull request merged**